src/test/java/bio/terra/service/dataset/DatasetBucketDaoTest.java

```diff
@@ -81,7 +81,8 @@ public void setup() throws IOException {
     projectResource.id(projectId);
     projectIds.add(projectId);
 
-    dataset = daoOperations.createMinimalDataset(billingProfile.getId(), projectId);
+    dataset =
+        daoOperations.createDataset(billingProfile.getId(), projectId, "dataset-minimal.json");
```

Comment:
It's probably not worth doing for this PR, but I wonder if there's a way to combine the two test datasets so there's only one that covers all cases being tested.